### DISCUSSION TITLE
What will be the clever way to save emails to Trilium?

### DISCUSSION THREAD

**Discussion by WantToLearnJapanese** [TriliumNext/Trilium#2934] [Q&A] [answered]
What will be the clever way to save emails to Trilium?

Just copy and paste would be the best way?

**Comment by zadam** [comment, marked as answer]
The cleverest would be to use IMAP to download mail with the help of your favorite programming language and use [ETAPI](https://github.com/zadam/trilium/wiki/ETAPI) to import them into Trilium.

Otherwise, perhaps if your email supports bulk export to HTML then these could be imported as well.
